### DISCUSSION TITLE
Página não encontrada - Trilha Python

### DISCUSSION THREAD

**Discussion by LuizFernando-TC** [codaqui/institucional#50] [Trilhas de Ensino]
O tópico "22 - Gerenciamento de Memória em Python" da trilha de Python possui um link de uma página que não existe, sendo portanto, necessário substitui-lo.
O primeiro link do tópico não está funcionando.

**Comment by endersonmenezes** [comment]
Boa @LuizFernando-TC vou converter em uma Issue e você pode cuidar desse assunto.

https://github.com/codaqui/institucional/issues/52

**Comment by endersonmenezes** [comment]
Solução enviada em: https://github.com/codaqui/institucional/issues/52
